server/jobs/soda_import_worker.py

```diff
@@ -25,8 +25,8 @@
 
 #: geojson of state boundaries source url
 states_source = (
-    'https://gist.githubusercontent.com/jbeezley/82150e34cee8512815cf/'
-    'raw/32549660c18149698a849b00cac6df8ddd2d1f1a/us-states.json'
+    'https://gist.githubusercontent.com/jbeezley/ec21c4a016a84c5def74/'
```

Comment:
If you really want me to, I will.  I was hesitant to do so because it is 2.5 Mb.  I don't like committing files that large to git.  Plus, there will almost certainly be a need for other similar datasets that will definitely be too large to commit.  I'm not sure how we want to handle this problem when it comes to that.